.github/scripts/version_check.py

```diff
@@ -18,6 +18,9 @@
 
 import requests
 
+REPO = os.getenv('GITHUB_REPOSITORY', 'inventree/inventree')
+GITHUB_API_URL = os.getenv('GITHUB_REPOSITORY', 'https://api.github.com')
```

Comment:
These are both accessing the same environmental variable - is this intentional?

Reply:
No, that is a typo. Good catch!